editors/vscode/tests/WorkspaceConfig.spec.ts

```diff
@@ -34,31 +46,21 @@ suite('WorkspaceConfig', () => {
     strictEqual(config.tsConfigPath, null);
     strictEqual(config.unusedDisableDirectives, 'allow');
     strictEqual(config.typeAware, false);
-    deepStrictEqual(config.flags, {});
+    strictEqual(config.disableNestedConfig, false);
+    strictEqual(config.fixKind, "safe_fix");
     strictEqual(config.formattingExperimental, false);
     strictEqual(config.formattingConfigPath, null);
   });
 
-  test('configPath defaults to null when using nested configs and configPath is empty', async () => {
-    const wsConfig = workspace.getConfiguration('oxc', WORKSPACE_FOLDER);
-    await wsConfig.update('configPath', '', ConfigurationTarget.WorkspaceFolder);
-    await wsConfig.update('flags', {}, ConfigurationTarget.WorkspaceFolder);
+  test('deprecated values are respected', async () => {
+    await updateConfiguration('flags', {
+      disable_nested_config: 'true',
+      fix_kind: 'dangerous_fix'
+    });
 
     const config = new WorkspaceConfig(WORKSPACE_FOLDER);
-
-    deepStrictEqual(config.flags, {});
-    strictEqual(config.configPath, null);
-  });
-
-  test('configPath defaults to .oxlintrc.json when not using nested configs and configPath is empty', async () => {
```

Comment:
These 2 test should be implementation details on the server. 
They area already `LintOptions::use_nested_config()` tests, so we can safely delete them